pkg/auth/providers/saml/saml_account.go

```diff
@@ -0,0 +1,8 @@
+package saml
+
+type Account struct {
+	UID         string `json:"uid,omitempty"`         //DN
```

Comment:
Why do we call this field UID? Does it have special meaning in the saml world? Can we just call it it ID or PrincipalId?
For that matter, do we even need this Account struct or can we go straight to the Principal struct? You may have been alluding to this yourself the other day.